mapviz_plugins/src/odometry_plugin.cpp

```diff
@@ -411,5 +467,10 @@ namespace mapviz_plugins
     emitter << YAML::Key << "static_arrow_sizes" << YAML::Value << ui_.static_arrow_sizes->isChecked();
 
     emitter << YAML::Key << "arrow_size" << YAML::Value << ui_.arrow_size->value();
+
+    emitter << YAML::Key << "show_timestamps" << YAML::Value << ui_.show_timestamps->value();
```

Comment:
This YAML key doesn't appear to have anything to do with showing all covariance ovals.

Please constrain changes in a PR to a single issue.